### PR TITLE
feat/DEVSU-2302 minor appendixview visual changes

### PR DESCRIPTION
**Pull request by sshugsc**
[DEVSU-2302](https://www.bcgsc.ca/jira/browse/DEVSU-2302)
* minor appendixview visual change

### CODE REVIEW

File: app/views/AdminView/components/Appendices/columnDefs.ts [hunk 1]
```diff
@@ -3,12 +3,12 @@ import { ColDef } from '@ag-grid-community/core';
 
 const columnDefs: ColDef[] = [
   {
-    headerName: 'Name',
+    headerName: 'Tamplate Name',
```
Comment (elewis2) [inline app/views/AdminView/components/Appendices/columnDefs.ts:6]: Tamplate -> Template

Comment (sshugsc) [reply app/views/AdminView/components/Appendices/columnDefs.ts:6]: oh sorry

File: app/views/AdminView/components/Appendices/columnDefs.ts [hunk 2]
```diff
@@ -3,12 +3,12 @@ import { ColDef } from '@ag-grid-community/core';
...
     field: 'template.name',
   },
   {
-    headerName: 'Org',
-    field: 'template.organization',
+    headerName: 'Project',
+    cellRenderer: ({ data }) => data.project.name || 'default appendix text',
```
Comment (elewis2) [inline app/views/AdminView/components/Appendices/columnDefs.ts:11]: Not sure this looks right. Maybe 'default appendix text' -> '(default appendix text)' and if that doesn't look right we'll figure something else out later

**Review by elewis2 — CHANGES_REQUESTED** [review]
see two comments - one spelling mistake, and the default text change

**Review by elewis2 — APPROVED** [review]

**Pull request merged**